CMakeLists.txt

```diff
@@ -138,12 +138,26 @@ option(GOOGLE_CLOUD_CPP_ENABLE_FIRESTORE
        "Enable building the Firestore library." ON)
 option(GOOGLE_CLOUD_CPP_ENABLE_PUBSUB "Enable building the Pub/Sub library." ON)
 
+# Compute the default value for GOOGLE_CLOUD_CPP_ENABLE_GRPC_UTILS, use a
+# function to create a scope for the temporary variables.
+function(google_cloud_cpp_grpc_utils_expression)
+    set(expression "GOOGLE_CLOUD_CPP_ENABLE_BIGQUERY")
+    foreach (lib BIGTABLE SPANNER PUBSUB)
+        string(APPEND expression " OR GOOGLE_CLOUD_CPP_ENABLE_${lib}")
+    endforeach()
+    set(GOOGLE_CLOUD_CPP_ENABLE_GRPC_UTILS_EXPRESSION "${expression}" PARENT_SCOPE)
+endfunction()
+google_cloud_cpp_grpc_utils_expression()
```

Comment:
One thought: This function might be more complicated than something like

```cmake
string(CONCAT GOOGLE_CLOUD_CPP_ENABLE_GRPC_UTILS_EXPRESSION
    "GOOGLE_CLOUD_CPP_ENABLE_BIGQUERY OR "
    "GOOGLE_CLOUD_CPP_ENABLE_BIGTABLE OR "
    "GOOGLE_CLOUD_CPP_ENABLE_SPANNER OR "
    "GOOGLE_CLOUD_CPP_ENABLE_PUBSUB"
)
```
and this version is more grep-able so we hopefully might not miss it the next time we're adding things.

Totally optional. Your call about which you prefer.

Reply:
I like, thanks!

BTW, `string(JOIN...)` would have been nice, but does not exist in CMake 3.5 ☹️